app/src/components/Modals/add-activity-modal.tsx

```diff
@@ -214,6 +216,11 @@ const AddActivityModal: FC<AddUserModalProps> = ({
                           placeholder={t("activity-data-amount-placeholder")}
                           borderRightRadius={0}
                           bgColor="base.light"
+                          _focus={{
+                            borderWidth: "1px",
+                            shadow: "none",
+                            borderColor: "content.link",
+                          }}
```

Comment:
These styles can potentially be moved to the app theme, so we can apply them as the default for most form inputs. Fine for now though.